cmd/cloud/cluster.go

```diff
@@ -473,7 +476,7 @@ var clusterListCmd = &cobra.Command{
 					cluster.State,
 					cluster.ProvisionerMetadataKops.Version,
 					fmt.Sprintf("%d x %s", cluster.ProvisionerMetadataKops.MasterCount, cluster.ProvisionerMetadataKops.MasterInstanceType),
-					fmt.Sprintf("%d x %s (max %d)", cluster.ProvisionerMetadataKops.NodeMinCount, cluster.ProvisionerMetadataKops.NodeInstanceType, cluster.ProvisionerMetadataKops.NodeMaxCount),
+					fmt.Sprintf("%d x %s (max %d) %s", cluster.ProvisionerMetadataKops.NodeMinCount, cluster.ProvisionerMetadataKops.NodeInstanceType, cluster.ProvisionerMetadataKops.NodeMaxCount, cluster.ProvisionerMetadataKops.Networking),
```

Comment:
If we want to show the networking value in this table then it should probably be added as a new column and separated out from the node count.

Reply:
Agreed, will look into it.

Reply:
Added networking value as a separate column, now it looks as follows.
<img width="757" alt="Screen Shot 2021-03-29 at 9 50 17 am" src="https://user-images.githubusercontent.com/53712140/112770934-fa9aff00-9074-11eb-8887-105288c4cb16.png">